### PR TITLE
[WEB-1255] chore: Refactor existing Space app for project publish

### PR DESCRIPTION
**Pull request by rahulramesha**
This PR includes Refactoring in Existing Project publish by introducing Pagination. And also include changes to make way for Advanced views in Space App.

https://github.com/makeplane/plane/assets/71900764/dded0e69-738c-4964-abac-2a69273a7792

### CODE REVIEW

File: space/core/components/issues/issue-layouts/list/header.tsx [hunk 1]
```diff
@@ -2,25 +2,32 @@
...
+  const { getGroupIssueCount } = useIssue();
+
+  const state = getStateById(stateId);
 
   return (
-    <div className="flex items-center gap-2 p-3">
+    <div className="flex sticky top-0 items-center gap-2 p-3 bg-custom-background-90 z-[1] border-b-[1px] border-custom-border-200">
       <div className="flex h-3.5 w-3.5 items-center justify-center">
-        <StateGroupIcon stateGroup={state.group} color={state.color} height="14" width="14" />
+        <StateGroupIcon stateGroup={state?.group ?? "backlog"} color={state?.color} height="14" width="14" />
```
Comment (SatishGandham) [inline space/core/components/issues/issue-layouts/list/header.tsx:25]: Can a state exist without group?

File: space/core/components/issues/peek-overview/issue-emoji-reactions.tsx [hunk 2]
```diff
@@ -71,17 +71,17 @@ export const IssueEmojiReactions: React.FC<IssueEmojiReactionsProps> = observer(
           const reactions = groupedReactions?.[reaction] ?? [];
           const REACTIONS_LIMIT = 1000;
 
-          if (reactions.length > 0)
+          if (reactions?.length > 0)
```
Comment (SatishGandham) [inline space/core/components/issues/peek-overview/issue-emoji-reactions.tsx:74]: Its always an array.

File: space/core/components/issues/peek-overview/issue-vote-reactions.tsx [hunk 3]
```diff
@@ -73,13 +73,13 @@ export const IssueVotes: React.FC<TIssueVotes> = observer((props) => {
       <Tooltip
         tooltipContent={
           <div>
-            {allUpVotes.length > 0 ? (
+            {allUpVotes?.length > 0 ? (
```
Comment (SatishGandham) [inline space/core/components/issues/peek-overview/issue-vote-reactions.tsx:76]: Isn't this always an array, and on line 79 to.

File: space/core/components/issues/peek-overview/issue-properties.tsx [hunk 4]
```diff
@@ -58,7 +67,7 @@ export const PeekOverviewIssueProperties: React.FC<Props> = ({ issueDetails, mod
             <span>State</span>
           </div>
           <div className="w-3/4 flex items-center gap-1.5 py-0.5 text-sm">
-            <StateGroupIcon stateGroup={state.group} color={state.color} />
+            <StateGroupIcon stateGroup={state?.group ?? "backlog"} color={state?.color} />
```
Comment (SatishGandham) [inline space/core/components/issues/peek-overview/issue-properties.tsx:70]: Shouldnt color also have a fallback?

**Review by SatishGandham — CHANGES_REQUESTED** [review]

**Review by SatishGandham — APPROVED** [review]

**Review by pablohashescobar — APPROVED** [review]

**Pull request merged**